src/simmi/assignment_15/Accounts.java

```diff
@@ -0,0 +1,170 @@
+/*A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI*/
+
+package simmi.assignment_15;
+
+import java.util.Scanner;
+
+public class Accounts {
+	int iciciBalance = 10000, hdfcBalance = 8000, bobBalance = 9000, sbiBalance = 5000;
+	int debitCountICICI = 0, debitCountHDFC = 0, debitCountBOB = 0, debitCountSBI = 0;
+	int creditCountICICI = 0, creditCountHDFC = 0, creditCountBOB = 0, creditCountSBI = 0;
+	
+	void operation(String bank, String operation, int amount) {
+		switch(operation) {
+		case "debit" :
+			switch(bank) {
+			case "ICICI" :
+				if(iciciBalance >= amount) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method